mmocr/datasets/utils/loader.py

```diff
@@ -39,6 +39,17 @@ def __getitem__(self, index):
         """Retrieve anno info of one instance with dict format."""
         return self.parser.get_item(self.ori_data_infos, index)
 
+    def __iter__(self):
+        self._n = 0
+        return self
+
+    def __next__(self):
+        if self._n < len(self):
+            data = self[self._n]
+            self._n += 1
+            return data
+        raise StopIteration
```

Comment:
it will always lead to stopiteration exception as self._n keeps increasing?

Reply:
yeah this is intended